src/runtime/hexagon_remote/Makefile

```diff
@@ -118,6 +125,10 @@ bin/%/libhalide_hexagon_remote_skel.so: bin/%/halide_remote.o bin/%/halide_hexag
 	-Wl,-Bsymbolic -Wl,--wrap=malloc -Wl,--wrap=calloc -Wl,--wrap=free \
 	-Wl,--wrap=realloc -Wl,--wrap=memalign -Wl,--wrap=__stack_chk_fail
 
+bin/%/signed_by_debug/libhalide_hexagon_remote_skel.so: bin/%/libhalide_hexagon_remote_skel.so
+	mkdir -p $(@D)
+	OPENSSL_DIR=/usr/bin /usr/bin/python $(HEXAGON_ELFSIGNER) --no_disclaimer -i $^ -o `dirname $@`
```

Comment:
I have no idea. It seems like the right thing to do is actually not have this path at all (or the fully qualified path on python), and let the user specify OPENSSL_DIR (outside the Make rule) if it's somewhere other than /usr/bin.